Week1/homework/exercise1/server.js

```diff
@@ -0,0 +1,133 @@
+let mysql = require("mysql");
+const employees = require("./employees");
+const departments = require("./departments");
+const projects = require("./projects");
+
+const queryTablesArray = [
+    "CREATE TABLE IF NOT EXISTS Employee(emp_no INT NOT NULL AUTO_INCREMENT, emp_name Varchar(25) NOT NULL, salary Double NOT NULL," +
+    "reports_to Varchar(25) NOT NULL, PRIMARY KEY(emp_no));",
+    "CREATE TABLE IF NOT EXISTS Departments(dept_no INT NOT NULL AUTO_INCREMENT, dept_name Varchar(25) NOT NULL," +
+    "manager Varchar(25) NOT NULL, PRIMARY KEY(dept_no));",
+    "CREATE TABLE IF NOT EXISTS Projects(proj_no INT NOT NULL AUTO_INCREMENT, proj_name Varchar(25) NOT NULL," +
+    "starting_date Date NOT NULL, ending_date Date NULL, PRIMARY KEY(proj_no));"
+];
+
+const messagesArray = [
+    "Table Employee successfully created",
+    "Table Departments successfully created",
+    "Table Projects successfully created"
+];
+
+function createDatabase(database_name){
+    connection = mysql.createConnection({
```

Comment:
you could've created the `connection` instance at the top level, and reused it in each function, instead of creating a different connection each time?